### PR TITLE
chore(infra): Deploy relay and portal to more zones and use more modern CPU arch

### PR DESCRIPTION
**Pull request by AndrewDryga**

### CODE REVIEW

File: website/src/app/kb/architecture/tech-stack/readme.mdx [hunk 1]
```diff
@@ -106,19 +106,20 @@ Firezone uses the following tools for ops and infrastructure:
...
+| Saint-Ghislain, Belgium            | `europe-west1`         | "europe-west1-b", "europe-west1-c", "europe-west1-d"                                |
+| London, UK                         | `europe-west2`         | "europe-west2-a", "europe-west2-b", "europe-west2-c"                                |
+| São Paulo, Brazil                  | `southamerica-east1`   | "southamerica-east1-a", "southamerica-east1-b", "southamerica-east1-c"              |
+| Council Bluffs, Iowa, USA          | `us-central1`          | "us-central1-a", "us-central1-b", "us-central1-c", "us-central1-d", "us-central1-f" |
+| Moncks Corner, South Carolina, USA | `us-east1`             | "us-east1-a", "us-east1-b", "us-east1-c", "us-east1-d"                              |
+| Los Angeles, California, USA       | `us-west2`             | "us-west2-a", "us-west2-b", "us-west2-c"                                            |
+| Ashburn, Northern Virginia, USA    | `us-east4`             | "us-east4-a", "us-east4-b", "us-east4-c"                                            |
```
Comment (jamilbk) [inline website/src/app/kb/architecture/tech-stack/readme.mdx]:
```suggestion
| Changhua, Taiwan                   | `asia-east1`           | `asia-east1-a`, `asia-east1-b`, `asia-east1-c`                                      |
| Mumbai, India                      | `asia-south1`          | `asia-south1-a`, `asia-south1-b`, `asia-south1-c`                                   |
| Sydney, Australia                  | `australia-southeast1` | `australia-southeast1-a`, `australia-southeast1-b`, `australia-southeast1-c`        |
| Warsaw, Poland                     | `europe-central2`      | `europe-central2-a`, `europe-central2-b`, `europe-central2-c`                       |
| Hamina, Finland                    | `europe-north1`        | `europe-north1-a`, `europe-north1-b`, `europe-north1-c`                             |
| Saint-Ghislain, Belgium            | `europe-west1`         | `europe-west1-b`, `europe-west1-c`, `europe-west1-d`                                |
| London, UK                         | `europe-west2`         | `europe-west2-a`, `europe-west2-b`, `europe-west2-c`                                |
| São Paulo, Brazil                  | `southamerica-east1`   | `southamerica-east1-a`, `southamerica-east1-b`, `southamerica-east1-c`              |
| Council Bluffs, Iowa, USA          | `us-central1`          | `us-central1-a`, `us-central1-b`, `us-central1-c`, `us-central1-d`, `us-central1-f` |
| Moncks Corner, South Carolina, USA | `us-east1`             | `us-east1-a`, `us-east1-b`, `us-east1-c`, `us-east1-d`                              |
| Los Angeles, California, USA       | `us-west2`             | `us-west2-a`, `us-west2-b`, `us-west2-c`                                            |
| Ashburn, Northern Virginia, USA    | `us-east4`             | `us-east4-a`, `us-east4-b`, `us-east4-c`                                            |
```

File: terraform/environments/production/relays.tf [hunk 2]
```diff
@@ -47,49 +47,56 @@ module "relays" {
...
+      replicas   = 1
+      zones      = ["us-east1-a", "us-east1-b", "us-east1-c", "us-east1-d"]
     }
 
     "us-west2" = {
       cidr_range = "10.137.0.0/24"
-      type       = "f1-micro"
-      replicas   = 2
-      zones      = ["us-west2-b"]
+      type       = "n2-standard-2"
```
Comment (jamilbk) [inline terraform/environments/production/relays.tf:69]: What if we use `e2-micro` instead? `n2-standard-2` is about 15x the cost of `f1-micro` according to https://gcloud-compute.com/instances.html

`e2-micro` seems to be available in all regions as well.

Comment (AndrewDryga) [reply terraform/environments/production/relays.tf:69]: @jamilbk you can not reserve them, the whole purpose of this change is to have something that will be always available. I've switched them to `highcpu` variant with is about half the price.

**Comment by AndrewDryga** [comment]
Merging as it's deployed manually.

**Pull request merged**